internal/ingest/goa.go

```diff
@@ -177,56 +175,22 @@ func (svc *ingestImpl) ListSipWorkflows(
 		return nil, goaingest.MakeNotValid(errors.New("invalid UUID"))
 	}
 
-	s, err := svc.perSvc.ReadSIP(ctx, sipUUID)
-	if err == sql.ErrNoRows {
+	if _, err := svc.perSvc.ReadSIP(ctx, sipUUID); err == persistence.ErrNotFound {
 		return nil, &goaingest.SIPNotFound{UUID: payload.UUID, Message: "SIP not found"}
 	} else if err != nil {
 		return nil, goaingest.MakeNotAvailable(errors.New("cannot perform operation"))
```

Comment:
Not for this PR, but we are starting to return internal errors (500) from these cases.